Marlin/Makefile

```diff
@@ -350,11 +350,11 @@ LDFLAGS = -lm
 AVRDUDE_PORT = $(UPLOAD_PORT)
 AVRDUDE_WRITE_FLASH = -Uflash:w:$(BUILD_DIR)/$(TARGET).hex:i
 ifeq ($(shell uname -s), Linux)
-AVRDUDE_CONF = $(ARDUINO_INSTALL_DIR)/hardware/tools/avrdude.conf
+AVRDUDE_CONF = /etc/avrdude/avrdude.conf
```

Comment:
While this applies fine to Linux it doesn't apply to other platforms, such as OS X. So instead of explicitly defining these paths, the Makefile should allow us to set an environment var such as $AVRDUDE at the top.